kbase-extension/static/kbase/js/widgets/appWidgets2/view/structView.js

```diff
@@ -40,80 +41,64 @@ define([
             viewModel.state.enabled = false;
         }
 
-        function setModelValue(value) {
-            return Promise.try(() => {
```

Comment:
This is only called internally, no need for Promises that don't do anything async.